### PR TITLE
Fix errors in main.py

### PR DESCRIPTION
**Pull request by ephraimberkovitch**
Simple syntax errors and misspellings

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -283,6 +283,10 @@ def log_bad_request(request):
     except AttributeError:
         logging.debug("Bad request:{0}".format(str(request)))
 
+@app.route("/test")
```
Comment (danielhers) [inline main.py]: Why is this `/test` route necessary?

File: main.py [hunk 2]
```diff
@@ -470,7 +474,7 @@ class HistoricalReportPeriods:
     def __init__(self,period_id, period_value, period_string):
         self.period_id=period_id
         self.period_value=period_value
-        self.severity_string=severity_string
+        self.severity_string=period_string
```
Comment (danielhers) [inline main.py]: I would fix the function argument name to `severity_string` instead.

File: main.py [hunk 3]
```diff
@@ -809,7 +813,7 @@ def create_years_list():
         try:
             year_col = db.session.query(distinct(func.extract("year", Marker.created)))
             break
-        except OperationalError:
+        except:
```
Comment (danielhers) [inline main.py]: It's generally preferable to be specific in `except` clauses. Why not add the appropriate `import`?

File: main.py (hunk 1 above)
Comment (ephraimberkovitch) [inline main.py]: It is not necessary, of course. I used just to ensure that all the endpoints function correctly. You want me to remove it and re-submit the pull request?

Comment (danielhers) [inline main.py]: Please do - it's ok for testing locally but there's no reason to have it in the website.
Also please see my other comments.

File: main.py (hunk 3 above)
Comment (ephraimberkovitch) [inline main.py]: OperationalError is a generic MySQL exception (_mysql_exceptions). May be, it would be better to replace it by SQLAlchemy exceptions?

Comment (danielhers) [inline main.py]: I remember it was just throwing an `OperationalError` sometimes due to a locked DB, which is why I wrote this loop to retry the query until success. I don't think it was an SQLAlchemy exception.

**Comment by ephraimberkovitch** [comment]
Would you like me to go over the source code and consider replacing SQLite by some cloud relational DB, like Postgres available as an add-on on Heroku?

**Comment by danielhers** [comment]
We already use Postgres. Nothing in the code assumes SQLite. The `DATABASE_URL` determines which DB to use. We are using the Heroku Postgres add-on, but we're in the process of switching to DigitalOcean, where we will also use Postgres.

**Comment by ephraimberkovitch** [comment]
I thought you are using SQLite because of the following line in the GitHub README.md file:

export DATABASE_URL='sqlite:///local.db'

Another question - if all the data is stored in PostgreSql, how do DB locks may appear?

**Comment by danielhers** [comment]
We are using SQLite when developing locally because it's easier than to set up a Postgres DB, but on the server we're using Postgres.
The lock will only happen with SQLite.

**Comment by ephraimberkovitch** [comment]
Got it. So I may keep using

from sqlite3 import OperationalError

at the moment?

**Comment by danielhers** [comment]
Only I think you were right and it should be `from sqlalchemy.exc import OperationalError` because SQLAlchemy will wrap this exception...

**Pull request merged**